examples/deploy-virtual-appliance-from-marketplace/README.md

```diff
@@ -0,0 +1,30 @@
+# Deploy a virtual appliance VM from Azure Marketplace
+
+deploy a VM from Azure Marketplace. In order to enable deployment in a programatic way (such as with Terrafrom), you first need to enable it in Azure. This is a one time action per Subscription per Solution.
```

Comment:
This might read better as:
```suggestion
In order to enable deployment in a programatic way (such as with Terrafrom), you first need to enable it in Azure. This is a one time action per Subscription per Solution.
```